app/src/components/items/overview/citations/overview.tsx

```diff
@@ -90,14 +91,12 @@ ${metadata.origin ? "(" + metadata.origin + ")" : ""}${
           const value = citationFormatDummyData[field];
           if (value !== "") {
             return (
-              <>
-                <Text key={index} size="overline1" marginBottom="xs">
+              <Fragment key={`citation-${field}`}>
```

Comment:
curious about why this needs a Fragment?

Reply:
I'll read the docs

Reply:
`<>` is just shorthand for a `Fragment` - if you want to add any props to a `<>` though, you have to explicitly use `Fragment`